CodeiumVS/LanguageServer/LanguageServer.cs

```diff
@@ -738,10 +738,10 @@ private async Task InitializeTrackedWorkspaceAsync()
         {
             try
             {
-                string projectDir = Path.GetDirectoryName(project.FullName);
-                await _package.LogAsync($"Project Dir: {projectDir}");
-                if (!string.IsNullOrEmpty(projectDir))
+                if (!string.IsNullOrEmpty(project.FullName))
                 {
+                    string projectDir = Path.GetDirectoryName(project.FullName);
+                    await _package.LogAsync($"Project Dir: {projectDir}");
                     AddTrackedWorkspaceResponse response = await AddTrackedWorkspaceAsync(projectDir);
                     if (response != null)
                     {
```

Comment:
Should we log the project.FullName as well? Seeing as that's actually what's being used now?

```suggestion
                    await _package.LogAsync($"Project FullName: {project.FullName}");
```

<sub>_[Suggestion by Codeium](https://codeium.com/forge)_</sub>